### PR TITLE
Admin can see a list of participants within a department

### PR DESCRIPTION
**Pull request by Fraseer**
PT Link: https://www.pivotaltracker.com/story/show/180222571

## User Story
As a HR department
In order to see who is participating from each department
I would like to see a table of participants

## Changes Proposed
- Create new view showing the participants within a department
- Add datagrid with the follow columns: name , total pnts, 

## Screenshots
image
image

### CODE REVIEW

File: cypress/integration/userCanSeeListOfParticipants.feature.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        cy.contains("Production").click();
+      });
+      cy.get(".MuiDataGrid-virtualScrollerRenderZone")
+        .children()
+        .should("have.length", 2);
+    });
+  });
+
+  describe("unsuccessfully", () => {
+    it("is expected that a toast will be displayed", () => {
```
Comment (maxarvid) [inline cypress/integration/userCanSeeListOfParticipants.feature.js]: Is there a toast for this or are we just displaying an empty department?

File: src/App.jsx [hunk 2]
```diff
@@ -8,16 +8,11 @@ import { ThemeProvider, createTheme } from "@mui/material/styles";
...
+import ParticipantList from "./components/ParticipantList";
 
 const App = () => {
   const { authenticated } = useSelector((state) => state);
-
-  if (navigator.language.includes("sv")) {
-    i18n.changeLanguage("sv");
-    store.dispatch({ type: "SET_LANGUAGE", payload: "sv" });
-  }
+  const { participantList } = useSelector((state) => state);
```
Comment (maxarvid) [inline src/App.jsx]: Can this be combined with line 15?

File: src/App.jsx [hunk 3]
```diff
@@ -8,16 +8,11 @@ import { ThemeProvider, createTheme } from "@mui/material/styles";
...
 import Login from "./components/Login";
 import { useSelector } from "react-redux";
-import store from "./state/store/configureStore";
-import i18n from "./i18n";
+import ParticipantList from "./components/ParticipantList";
 
 const App = () => {
   const { authenticated } = useSelector((state) => state);
-
-  if (navigator.language.includes("sv")) {
```
Comment (maxarvid) [inline src/App.jsx:17]: Does the navigation test still go green without this code?

File: src/components/ParticipantList.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { DataGrid } from "@mui/x-data-grid";
+import Container from "@mui/material/Container";
+import { useSelector } from "react-redux";
+
+const ParticipantList = () => {
+  const { participantList } = useSelector((state) => state);
+  const { departments } = useSelector((state) => state);
```
Comment (maxarvid) [inline src/components/ParticipantList.jsx]: Can line 7 and 8 be combined?

File: src/components/ParticipantList.jsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      flex: 1,
+    },
+    {
+      field: "score",
+      headerName: "Fika Score",
+      flex: 1,
+    },
+  ];
+
+  let rows = departments[participantList.id - 1].participants.map((participant) => {
```
Comment (maxarvid) [inline src/components/ParticipantList.jsx]: If `participantList` is a list, how can you call `.id` on it? I'm not sure what is going on here

File: cypress/integration/userCanSeeListOfParticipants.feature.js (hunk 1 above)
Comment (Fraseer) [reply cypress/integration/userCanSeeListOfParticipants.feature.js]: Originally I was going to, but then realised it wasnt needed I will change it

File: src/App.jsx (hunk 3 above)
Comment (Fraseer) [reply src/App.jsx:17]: It does, this is left over from trying to read the browsers default language and should have been removed before

File: src/App.jsx (hunk 2 above)
Comment (Fraseer) [reply src/App.jsx]: It can

File: src/components/ParticipantList.jsx (hunk 4 above)
Comment (Fraseer) [reply src/components/ParticipantList.jsx]: They can

File: src/components/ParticipantList.jsx (hunk 5 above)
Comment (Fraseer) [reply src/components/ParticipantList.jsx]: Maybe not the best naming, it'll be easier to show you than type it out

**Review by maxarvid — APPROVED** [review]
LGTM

**Review by AugustEnhager — APPROVED** [review]

**Pull request merged**